README.md

```diff
@@ -68,7 +68,14 @@ x86_64 = true
 
 ### Signature Help
 
-![](https://github.com/bergercookie/asm-lsp/blob/master/demo/signature.gif)
+![](https://github.com/bergercookie/asm-lsp/blob/master/demo/signaturehelp.gif)
+
+- Triggering signature help is dependent on your editor and LSP client.
+  - Using Neovim's built in LSP client, this can be done via the command
+    ``:lua vim.lsp.buf.signature_help()``.
```

Comment:
Out of curiosity, why the double backticks ? I think single ones would also do the job and format it as code.

Reply:
No real reason, I think double was required for code blocks on a messaging service I was using and I forgot to switch. I'll switch them over to single backticks and change the title of the `Demo` section `Demos / Features Documentation` once the pending PRs get cleared :)

Reply:
>  I think double was required for code blocks on a messaging service

Yeah, I often get confused as well 😓
I think [`sphinx`](https://www.sphinx-doc.org/) requires double backticks and I always get it mixed up